frontend/src/utils/validation.ts

```diff
@@ -10,6 +8,14 @@ export const namePattern = {
   message: "Invalid name",
 }
 
+export const isoDateTimePattern = {
```

Comment:
We have Zod in this project. Its a fantastic library for validation. There is probably something useful here https://zod.dev/?id=datetimes

Reply:
Thank you for mentioning zod library. I refactored it to use zod schema, which is more flexible than regex pattern.